src/Resources/views/AmazonPay/addressBookWidget.html.twig

```diff
@@ -10,7 +6,7 @@
     }
     var accessToken = getURLParameter("access_token", location.hash);
```

Comment:
Please, move this logic to controller. It's hard to look at this JS function when we've got proper tools for accessing data from the URL.